WordPress/gutenberg
# Font Library: Avoid running init functions when font library is available in core.

## What?
Avoid running init functions from the plugin when the font library is available in core.

## Why?
- Avoid creating repeated endpoints.
If the Gutenberg plugin is installed on WordPress core trunk (6.5-alpha) the font collection endpoints are created twice each, one by core and one by the plugin.  This PR avoids that.

## How?
Checking the WordPress version before running the endpoints creation function.
If the WordPress core version is < 6.5-alpha, the font collection endpoints are not created by the Gutenberg plugin.

## Testing Instructions
1. 
- Install and activate this Gutenberg PR on **WordPress core trunk version**.
- Check that the `/font-collections/` and `/font-collections/<slug>` endpoints are not repeated.
- Check that the font collections load as expected.

2. 
- Install and activate this Gutenberg PR on **WordPress 6.4.3 version**.
- Check that the `/font-collections/` and `/font-collections/<slug>` endpoints are not repeated.
- Check that the font collections load as expected.
